lib/node_modules/@stdlib/stats/base/dcumax/benchmark/benchmark.js

```diff
@@ -40,14 +47,9 @@ var dcumax = require( './../lib/dcumax.js' );
 */
 function createBenchmark( len ) {
 	var y;
```

Comment:
Don't do this sort of mixed declaration and assignment. In this case, it is fine to do

```
var x = uniform(...);
var y = new Float64Array( len );
```

For more complex scenarios, declare first and then assign later.